appengine/flexible/memcache/main.py

```diff
@@ -16,16 +16,26 @@
 import os
 
 from flask import Flask
-from pymemcache.client.base import Client as MemcacheClient
-
+import pylibmc
 
 app = Flask(__name__)
 
 
 # [START client]
-memcache_addr = os.environ.get('GAE_MEMCACHE_HOST', 'localhost')
-memcache_port = os.environ.get('GAE_MEMCACHE_PORT', 11211)
-memcache_client = MemcacheClient((memcache_addr, int(memcache_port)))
+# Environment variables are defined in app.yaml.
+if os.environ.get('USE_GAE_MEMCACHE'):
+    MEMCACHE_SERVER = ':'.join(
+        os.environ.get('GAE_MEMCACHE_HOST', 'localhost'),
+        os.environ.get('GAE_MEMCACHE_PORT', 11211))
```

Comment:
why not just check these env vars instead of having a separate env var as a flag?